server.py

```diff
@@ -301,6 +301,31 @@ def handle_tile(z, x, y, fmt, tile_pixel_size=None):
         return "Metatile fetch problem", 500
 
 
+@tile_bp.route('/tilezen/vector/v1/<int:tile_pixel_size>/all/tiles.<fmt>.json')
```

Comment:
Can the path include `tilejson` in it, please? Like : `tilejson.<fmt>.json`? That will improve SEO I think.

(Here and next line)